packages/react-components/src/types/VideoGalleryParticipant.ts

```diff
@@ -90,4 +90,10 @@ export interface VideoGalleryRemoteParticipant extends VideoGalleryParticipant {
   isSpeaking?: boolean;
   /** Video stream of shared screen */
   screenShareStream?: VideoGalleryStream;
+  /* @conditional-compile-remove(PSTN-calls) */
+  /** State that the participant is in */
```

Comment:
Let's add example states to the doc here just to make it more clear

Reply:
+1 to adding more doc comments here. `state` is a really vague term, state of what? e.g. it could randomly mean there video in a black and white state, or isSpeaking variable above could easily be considered a piece of "state" pertaining to this VideoGalleryRemoteParticipant. I can't think of a better name - perhaps connectedState? Otherwise provide a good doc comment on what state refers to here